doc/source/tune/_tutorials/tune-60-seconds.rst

```diff
@@ -9,54 +9,51 @@ Let's quickly walk through the key concepts you need to know to use Tune. In thi
     :local:
```

Comment:
this page needs to include TONS of links to other docs. Carpet bomb the document so that people can know where to look for more information.